packages/web-components/src/components/va-date/test/va-date.e2e.ts

```diff
@@ -7,7 +7,14 @@ describe('va-date', () => {
     await page.setContent('<va-date></va-date>');
 
     const element = await page.find('va-date');
+    const monthInput = await page.find('va-date >>> va-select.select-month')
+    const dayInput = await page.find('va-date >>> va-select.select-day')
+    const yearInput = await page.find('va-date >>> va-text-input.input-year')
+
     expect(element).toHaveClass('hydrated');
+    expect(monthInput).not.toBeNull();
+    expect(dayInput).not.toBeNull();
+    expect(yearInput).not.toBeNull();
```

Comment:
I added these expectations so that we know that the `expect(dayInput).toBeNull();` down in the `monthYearOnly` tests isn't a false positive.

Reply:
Could you add a separate test for this then, so that we know what we're looking for if it breaks?